### ISSUE TITLE
Standard Reports Ignoring User Permissions

### ISSUE THREAD

**Issue by moizsami** [frappe/erpnext#36930]
### Information about bug

Hello,

I have set multiple user permissions for each user in my system, such as on customer group, price list, etc..

When I try to access certain reports such as 'Inactive Customers' from a restricted user, the user is able to see details for other customer groups as well, Which is not correct.

For Example if User 'Adam' is restricted to 'East Coast' customer group, in the reports it should only display data of customers who are part of 'East Coast' customer group, instead it is showing all customer's data.

Since I am using Frappe Cloud, I am unable to modify the report. Need help in solving these issues with such reports.

Regards,

### Module

accounts, buying, selling

### Version

Frappe Framework: v14.44.0
ERPNext: v14.34.3

### Installation method

FrappeCloud

**Comment by shoaib0300** [comment]
@moizsami Check the role permissions assigned to the 'Adam' user to ensure that there are no roles with conflicting permissions that might be affecting data access.

Since you are using FrappeCloud, you can reach out to their support team to investigate the issue further. They may be able to assist in diagnosing and resolving the problem.

**Comment by naqeeb0098** [comment]
> ### Information about bug
> Hello,
> 
> I have set multiple user permissions for each user in my system, such as on customer group, price list, etc..
> 
> When I try to access certain reports such as 'Inactive Customers' from a restricted user, the user is able to see details for other customer groups as well, Which is not correct.
> 
> For Example if User 'Adam' is restricted to 'East Coast' customer group, in the reports it should only display data of customers who are part of 'East Coast' customer group, instead it is showing all customer's data.
> 
> Since I am using Frappe Cloud, I am unable to modify the report. Need help in solving these issues with such reports.
> 
> Regards,
> 
> ### Module
> accounts, buying, selling
> 
> ### Version
> Frappe Framework: v14.44.0 ERPNext: v14.34.3
> 
> ### Installation method
> FrappeCloud
> 
> ### Relevant log output / Stack trace / Full Error Message.
> _No response_

yes i have same issue i have also restrict my customer to specific sales person but when he open the reports he willa able to see data of all sales mans
how can i fix this issue

**Comment by mihir-kandoi** [comment]
This was due to query builder queries not checking for permission. It has now been fixed.
